packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -113,32 +114,30 @@ class MemoryController {
 
   Future computeLibraries() async {
     if (libraryCollection == null) {
-      final VM vm = await serviceManager.service.getVM();
-      // TODO(terry): Need to handle a possible Sentinel being returned.
-      final List<Isolate> isolates =
-          await Future.wait(vm.isolates.map((IsolateRef ref) async {
-        return await serviceManager.service.getIsolate(ref.id);
-      }));
-
-      libraryCollection = LibraryCollection(libraryFilters);
-      for (LibraryRef libraryRef in isolates[0].libraries) {
-        final Library theLibrary =
-            await serviceManager.service.getObject(_isolateId, libraryRef.id);
-        libraryCollection.addLibrary(theLibrary);
-      }
+      unawaited(serviceManager.service.getVM().then((vm) {
```

Comment:
whyis this one unawaited? Seems fine to wait for the VM to be available. As is the computeLibrary method seems misleading as it returns a Future that completes well before the libraries are completed.

Reply:
I agree the analyzer returned warning that an unwaited was needed I'm happy to override the warning because I think its a bad warning.

info: `Future` results in `async` function bodies must be `await`ed or marked `unawaited` using `package:pedantic`. (unawaited_futures at [devtools] lib/src/memory/memory_controller.dart:118)